joblib/test/test_numpy_pickle.py

```diff
@@ -230,27 +200,27 @@ def test_numpy_persistence():
 
 
 @with_numpy
-def test_numpy_persistence_bufferred_array_compression():
+def test_numpy_persistence_bufferred_array_compression(tmpdir):
     big_array = np.ones((_IO_BUFFER_SIZE + 100), dtype=np.uint8)
-    filename = env['filename'] + str(random.randint(0, 1000))
+    filename = tmpdir.join('test.pkl').strpath + str(random.randint(0, 1000))
```

Comment:
@lesteve there is a session scoped `tmpdir_factory` fixture available as well. I can make a subdirectory, and limit it to a module scoped if needed.